lib/ansible/modules/monitoring/zabbix/zabbix_screen.py

```diff
@@ -316,7 +353,16 @@ def main():
             http_login_password=dict(type='str', required=False, default=None, no_log=True),
             validate_certs=dict(type='bool', required=False, default=True),
             timeout=dict(type='int', default=10),
-            screens=dict(type='list', required=True)
+            screens=dict(type='list', elements='dict', required=True, options=dict(
+                screen_name=dict(type='str', required=True),
+                host_group=dict(type='str'),
+                state=dict(type='str', default='present', choices=['absent', 'present']),
+                graph_names=dict(type='list', elements='str'),
+                graph_width=dict(type='int', default=200),
```

Comment:
The default values of `graph_width` and `graph_height` in the code used to be `None`, not 200 resp. 100. Is this change intended? I don't know how the settings work internally (didn't bother to check), but such a change could modify the behavior of the module (if `None` was "keep whatever the user currently had configured").

Reply:
I guess it may cause an unexpected behaviour for users who already using it.
So, we should use `None`.